newsfragments/960.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+As a result of fixing a bug in ``trio.Path.home``, the two ``trio.Path``
```

Comment:
Use Sphinx markup to link to the corresponding section of the documentation: that is, say
```
[...] a bug in :meth:`trio.Path.home`, the two :class:`trio.Path` classmethods,
:meth:`~trio.Path.home` and :meth:`~trio.Path.cwd`, [...]
```
I think this might also read better as two sentences, something like: "The trio.Path classmethods, home and cwd, are now async functions. Previously, a bug in the forwarding logic meant cwd was synchronous and home didn't work at all.

Reply:
Nice suggestions.  Running ``towncrier --draft`` helped me see how things would look in the docs.